src/components/organisms/Features/features.module.css

```diff
@@ -0,0 +1,30 @@
+.featuresHeading {
+  text-align: center;
+  margin-bottom: var(--space-2xl);
+}
+
+.featuresItems {
+  list-style: none;
```

Comment:
Is there a reset anywhere? NBD if there's not for a site this size

Reply:
There is a reset, but this isn't something that we necessarily want to apply to ALL lists, so I would typically capture this as a mixin, but since this site isn't using Sass, and it probably won't be used very much, I just put it directly in here.